std_msgs/Int16MultiArray input
---
std_msgs/Float32MultiArray output